src/http/status/status.service.ts

```diff
@@ -4,34 +4,33 @@ import { APP_VERSION } from 'app/app.constants';
 import { Status } from './entities';
 import { Injectable } from '@nestjs/common';
 import { EntityManager } from '@mikro-orm/postgresql';
-import { CuratedModuleService } from 'staking-router-modules';
 import { ValidatorsService } from 'validators';
 import { IsolationLevel } from '@mikro-orm/core';
+import { StakingRouterService } from 'staking-router-modules/staking-router.service';
 
 @Injectable()
 export class StatusService {
   constructor(
-    protected readonly curatedService: CuratedModuleService,
     protected readonly validatorsService: ValidatorsService,
     protected readonly configService: ConfigService,
     private readonly entityManager: EntityManager,
+    private readonly stakingRouterService: StakingRouterService,
   ) {}
 
   public async get(): Promise<Status> {
     const chainId = this.configService.get('CHAIN_ID');
-
-    const { registryMeta, validatorsMeta } = await this.entityManager.transactional(
+    const { elMeta, clMeta } = await this.entityManager.transactional(
       async () => {
-        const registryMeta = await this.curatedService.getMetaDataFromStorage();
-        const validatorsMeta = await this.validatorsService.getMetaDataFromStorage();
+        const elMeta = await this.stakingRouterService.getElBlockSnapshot();
+        const clMeta = await this.validatorsService.getMetaDataFromStorage();
 
-        return { registryMeta, validatorsMeta };
+        return { elMeta, clMeta };
       },
       { isolationLevel: IsolationLevel.REPEATABLE_READ },
     );
 
-    const elBlockSnapshot = registryMeta ? new ELBlockSnapshot(registryMeta) : null;
-    const clBlockSnapshot = validatorsMeta ? new CLBlockSnapshot(validatorsMeta) : null;
+    const elBlockSnapshot = elMeta ? new ELBlockSnapshot(elMeta) : null;
```

Comment:
added task to return 425 if null https://linear.app/lidofi/issue/VAL-368/check-null-value-in-status-schema